samples/nrf9160/secure_boot/src/main.c

```diff
@@ -65,166 +65,195 @@
 extern void irq_target_state_set(unsigned int irq, int secure_state);
 extern int irq_target_state_is_secure(unsigned int irq);
 
-#define SECURE_FLASH_REGION_FIRST    0
-#define SECURE_FLASH_REGION_LAST     7
-#define NONSECURE_FLASH_REGION_FIRST 8
-#define NONSECURE_FLASH_REGION_LAST  31
-
-#define SECURE_SRAM_REGION_FIRST     0
-#define SECURE_SRAM_REGION_LAST      7
-#define NONSECURE_SRAM_REGION_FIRST  8
-#define NONSECURE_SRAM_REGION_LAST   31
+/* printk wrapper, to turn off logs when booting silently */
+#define PRINT(...)                                                             \
+	do {                                                                   \
+		if (!IS_ENABLED(CONFIG_SB_BOOT_SILENTLY)) {                    \
```

Comment:
I know this is a pet peeve but could all the `\` be aligned ^^

Reply:
Thanks, it is aligned in the [source](https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/ebf854979ef842cf29406dc4ef232822bb39e8ba/samples/nrf9160/secure_boot/src/main.c). GitHub messes that up :)

Reply:
Ah okei xD Yeah it's probably the +/- addition on the side weird tho 👍

Reply:
@lemrey , would we be able to use the logger, here?

Reply:
I avoided using the logger on purpose to keep the output as uncluttered as possible and the ROM footprint as small as possible. I also compared this approach to disabling the Console/UART driver combo and this resulted in considerably less ROM used.